Rework Next.js quick start guide (wizard) (#12291)
getsentry/sentry-docs: platform-includes/getting-started-install/javascript.nextjs.mdx

To install Sentry using the installation wizard, run the following command within your project:

```bash
npx @sentry/wizard@latest -i nextjs
```

> The wizard starts guiding you through the setup process, which also includes enabling the Sentry features listed above.

Reviewer: We would need to remove the part about features if we remove the features section above.

<Alert level="warning" title="Important">

This guide assumes you enable all Sentry features during the wizard setup and allow it to create an example page and route. If you make different choices, you can still follow along, but you may need to make adjustments as you proceed.

</Alert>

You can always add or remove config options manually later if needed. Though, the earlier you set these up, the better.

> <Expandable permalink title="What does the installation wizard change inside your application?">

Reviewer 2: ```suggestion
<Expandable title="What does the installation wizard change inside your application?">
```

> - Create config files with the default `Sentry.init()` calls for each runtime (node, browser, edge)

Reviewer: ```suggestion
- Create config files with the default `Sentry.init()` calls for all runtimes (Node.js, Browser, and Edge)
```

Author: updated 👍

To install Sentry using the installation wizard, run the following command within your project:

```bash
npx @sentry/wizard@latest -i nextjs
```

The wizard then guides you through the setup process, asking you to enable additional (optional) Sentry features for your application beyond error monitoring.

<Expandable title="Want to learn more about these features?">

> - [**Error Monitoring**](/product/issues) (always enabled): Automatically report errors,

Reviewer 3: I know Issues is our error monitoring product, but clicking on 'Error Monitoring' and being directed to the 'Issues' page might be confusing for those less familiar with Sentry. 

What do you think about having this anchor text say 'issues' and providing some context in the description?

e.g. "Issues (always enabled): Sentry's core error monitoring product that automatically reports errors . . ."

Author: Let's do as you suggested! This is also a good idea because we also write about this "Issues" page under "Need help locating the captured errors in your Sentry project?"

Updated it 👍

  uncaught exceptions, and unhandled rejections. If you have something that
  looks like an exception, Sentry can capture it.
- [**Tracing**](/product/tracing): Track software performance while seeing the
  impact of errors across multiple systems. For example, distributed tracing
  allows you to follow a request from the frontend to the backend and back.

> - [**Session Replay**](/product/explore/session-replay/web/getting-started/):

Author: Yes, good idea! 
Updated 👍

  Get to the root cause of an issue faster by viewing a video-like reproduction
  of what was happening in the user's browser before, during, and after the
  problem.

</Expandable>

This guide assumes that you enable all features and allow the wizard to create an example page and route. You can add or remove features at any time, but setting them up now will save you the effort of configuring them manually later.

<Expandable title="What does the installation wizard change inside your application?">

> - Create config files with the default `Sentry.init()` calls for all runtimes (Node.js, Browser, and Edge)

Reviewer 3: ```suggestion
- Creation of config files with the default `Sentry.init()` calls for all runtimes (Node.js, Browser, and Edge)
```

Author: I kept the verbs active (with an -s ending to match the expandable title) instead of using -ion. This sounds simpler and less formal to me -- what do you think? 

(also, same for similar suggestions below)

Reviewer 3: Ok, good call!

> - Add a Next.js instrumentation hook to your project (`instrumentation.ts`)

Reviewer 3: ```suggestion
- Addition of Next.js instrumentation hook to your project (`instrumentation.ts`)
```

> - Create or update your Next.js config with the default Sentry settings

Reviewer 3: ```suggestion
- Creation of or updates to your Next.js config with the default Sentry settings
```

> - Create `.sentryclirc` with an auth token to upload source maps (this file is automatically added to `.gitignore`)

Reviewer 3: ```suggestion
- Creation of a `.sentryclirc` with an auth token to upload source maps (this file is automatically added to `.gitignore`)
```

> - Add an example page and route to your application to help verify your Sentry setup

Reviewer 3: ```suggestion
- Addition of an example page and route to your application to help verify your Sentry setup
```

To install Sentry using the installation wizard, run the following command within your project:

```bash
npx @sentry/wizard@latest -i nextjs
```

The wizard then guides you through the setup process, asking you to enable additional (optional) Sentry features for your application beyond error monitoring.

> <Expandable title="Want to learn a bit more about these features?">

Reviewer 4: ```suggestion
<Expandable title="Want to learn more about these features?">
```

I would tend to leave the `a bit` away, but no strong feelings!

Author: Let's remove it 👍

- [**Error Monitoring**](/product/issues) (always enabled): Automatically report errors,
  uncaught exceptions, and unhandled rejections. If you have something that
  looks like an exception, Sentry can capture it.
- [**Tracing**](/product/tracing): Track software performance while seeing the
  impact of errors across multiple systems. For example, distributed tracing
  allows you to follow a request from the frontend to the backend and back.
- [**Session Replay**](/product/explore/session-replay/web/getting-started/):
  Get to the root cause of an issue faster by viewing a video-like reproduction
  of what was happening in the user's browser before, during, and after the
  problem.

</Expandable>

<Alert level="warning" title="Important">

This guide assumes that you enable all features and allow the wizard to create an example page and route. Keep this in mind as you proceed.

</Alert>

> You can always add or remove features manually later if needed. Though, the earlier you set these up, the better.

Reviewer 4: ```suggestion
You can always add or remove features manually later if needed, but the setup is easiest if features are enabled from the start.
```

Not 100% sure, but it seems a bit "unclear" why it is better (from a users point of view). Maybe this is a bit clearer?

Author: We could also remove the Alert ("This guide assumes that you enable all features ...") and instead write in basic text:

This guide assumes that you enable all features and allow the wizard to create an example page and route. You can add or remove features at any time, but setting them up now will save you the effort of configuring them manually later.

@mydea what do you think?

Reviewer 4: sounds good to me!

Author: Updated!
